File: 2020/test/day_11_test.exs
defmodule Day11Test do
  use ExUnit.Case

  import Day11
  import TestHelper

  describe "Day11" do
    test "part1" do
      assert part1(input("day_11_example_1")) == 37
      assert part1(input("day_11")) == 2265
    end

    test "part2" do
      assert part2(input("day_11_example_1")) == 26
      assert part2(input("day_11")) == 2045
    end
  end
end

File: 2024/day11/day11_test.go
package day11

import (
	"testing"

	"github.com/stretchr/testify/assert"
)

func TestPart1(t *testing.T) {
	assert.Equal(t, 55312, Part1("input_example_1"))
	assert.Equal(t, 197357, Part1("input"))
}

func TestPart2(t *testing.T) {
	assert.Equal(t, 234568186890978, Part2("input"))
}
